### ISSUE TITLE
Minecarts, Animals, Elytras, Underwater

### ISSUE THREAD

**Issue by nco2k** [caoimhebyrne/thirdpersonboating#1]
any chance you could create third-person mods for minecart driving, animal riding, elytra flying, and underwater diving? although it would probably be better to put everything in one mod.

**Comment by Eianex** [comment]
yes please, auto third person mod does not provide updates for 1.20.4/5/6 and we need them. Also the author said he will Not be making them

**Comment by caoimhebyrne** [comment]
Sure, I'll try to add something like this soon! I will make it into one mod, and probably rename this one into something.
